cmd/cycloid/middleware/organization.go

```diff
@@ -1,12 +1,52 @@
 package middleware
 
 import (
+	strfmt "github.com/go-openapi/strfmt"
+	"regexp"
+
 	"github.com/cycloidio/cycloid-cli/client/client/organization_workers"
 	"github.com/cycloidio/cycloid-cli/client/client/organizations"
 	"github.com/cycloidio/cycloid-cli/client/models"
 	"github.com/cycloidio/cycloid-cli/cmd/cycloid/common"
 )
 
+func (m *middleware) CreateOrganization(name string, canonical string) (*models.Organization, error) {
+
+	params := organizations.NewCreateOrgParams()
+
+	if canonical == "" {
+		re := regexp.MustCompile(`[^a-zA-z0-9_\-]`)
+		canonical = re.ReplaceAllString(name, "-")
+	}
+
+	body := &models.NewOrganization{
+		Name:      &name,
+		Canonical: &canonical,
+	}
+
+	params.SetBody(body)
+	err := body.Validate(strfmt.Default)
+	if err != nil {
+		return nil, err
```

Comment:
the `CreateOrganization` can return two errors:
1. For the `body.Validate`
1. For the actual API call

I think it could me meaningful to add some context to this errors :) 

```suggestion
		return nil, errors.Wrap(err, "unable to validate request body")
```